### ISSUE TITLE
Do not use github-sponsors dependency

### ISSUE THREAD

**Issue by mdudek** [jeerbl/webfonts-loader#133]
The github-sponsors dependency throws exception when executed as postInstall script: 

```
Error: Cannot find module '/c:\sources\git\xxx\node_modules\webfonts-loader/package.json'
Require stack:
- c:\Sources\git\xxx\node_modules\github-sponsors\lib\github-sponsors.cjs.js
- c:\Sources\git\xxx\node_modules\github-sponsors\bin\github-sponsors.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:831:15)
    at Function.Module._load (internal/modules/cjs/loader.js:687:27)
    at Module.require (internal/modules/cjs/loader.js:903:19)
    at require (internal/modules/cjs/helpers.js:74:18)
    at printDonationMessage (c:\Sources\git\xxx\node_modules\github-sponsors\lib\github-sponsors.cjs.js:304:23)
    at init (c:\Sources\git\xxx\node_modules\github-sponsors\lib\github-sponsors.cjs.js:385:5) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    'c:\\Sources\\git\\xxx\\node_modules\\github-sponsors\\lib\\github-sponsors.cjs.js',
    'c:\\Sources\\git\\xxx\\node_modules\\github-sponsors\\bin\\github-sponsors.js'
  ]
}
```

The authors of github-sponsors do not recommend to use this package anymore:

```
Do not use this package! Add funding to your package.json instead: https://docs.npmjs.com/cli/v7/configuring-npm/package-json#funding
```

**Comment by jeerbl** [comment]
Just updated `package.json`, waiting for build before publishing new patch version.

**Comment by jeerbl** [comment]
Fixed in v7.1.1 :)
